src/app/commons/behaviours/text.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+angular.module('wegas.behaviours.text', [])
+    .directive('textSelectable', function() {
+        return function(scope, element, attr) {
+            $('html').click(function(event) {
+                $('#tmp').remove();
+            });
+            $(element).click(function(event) {
+                event.stopPropagation();
+                if ($('#tmp').length) {
+                    $('#tmp').remove();
+                }
+                var clickText = $(this).text();
+                $('<textarea id="tmp" />').attr("readonly", "readonly").appendTo($(this)).val(clickText).focus().select();
```

Comment:
Why aren't you doing something like
`$('<textarea id="tmp" readonly="readonly">'+clickText+'</textarea>').appendTo($(this)).focus().select()`

Personnally, I think it could be more understandable with a button after link

![capture d ecran 2015-05-21 a 07 34 58](https://cloud.githubusercontent.com/assets/1135557/7742180/06318634-ff8c-11e4-97b5-f2ce5bdd0c12.png)